### DISCUSSION TITLE
(Suggestion) Custom Stickers (again)

### DISCUSSION THREAD

**Discussion by thatisjake** [tsunamayo/Starship-EVO#4460] [Suggestions]
I assume the current stickers are using vector art. With this, it could be assumed that we can create our own stickers using adobe illustrator. I think this should be a feature just like the custom colors we can have. The possibilities are endless, and I believe it would be a great addition.

**Comment by pinesh** [comment]
reposting your previous suggestions isn't a great idea. Github may not have a *bump* feature but suggestion clones isn't a great precedent.
Consider deleting your previous one.  
https://github.com/tsunamayo/Starship-EVO/discussions/4339

**Reply by thatisjake** [reply]
Didn't know I could delete them, Ill do so.
